cogs/make_applicant.py

```diff
@@ -0,0 +1,192 @@
+"""Contains cog classes for making a user into an applicant."""
+
+from collections.abc import Sequence
+from typing import Final
```

Comment:
Imports should be below the `__all__` declaration (apart from the `Sequence` import, which is used to type-hint the `__all__` declaration.